app/views/travel_requests/_title_card.html.erb

```diff
@@ -1,7 +1,7 @@
 <card size="full-width" class="title-card">
   <card-media>
     <lux-icon-base width="50" height="50" icon-color="rgb(65, 70, 78)">
-      <<%= request.travel_category_icon %> />
+      <<%= request.travel_category_icon %>></<%= request.travel_category_icon %>>
```

Comment:
Is this relates to IE support?

Reply:
Yes, this relates to IE11 support. IE11 complains when there are not full closing tags and it affects the layout of the page. Would it be better to split this into a fresh issue/PR?